testcontainers/src/core/image.rs

```diff
@@ -157,6 +162,7 @@ impl ImageArgs for () {
 pub struct RunnableImage<I: Image> {
     image: I,
     image_args: I::Args,
+    tty: bool,
```

Comment:
Is this necessary? Can't we access it through `image`?

Reply:
Ah, eagle eyes! 🦅  An artifact left from the other review change. It's gone :)